src/express/index.js

```diff
@@ -15,23 +16,65 @@ const articlesRouter = require(`./routers/articles`);
  *
  * @type {number}
  * @const
- * @default
+ * @default 8080
  */
 const PORT_NUMBER = 8080;
 
+/**
+ * Путь до папки со статикой public. Она будет будет полностью доступна с помощью express.static.
+ *
+ * @const
+ * @type {string}
+ * @default public
+ */
+const PATH_TO_PUBLIC_DIR = `public`;
+
+/**
+ * Путь до папки с шаблономи.
+ *
+ * @const
+ * @type {string}
+ * @default templates
+ */
+const PATH_TO_TEMPLATES_DIR = `templates`;
+
 /**
  * Создаем экземпляр Express.
  * @type {*|Express}
  */
 const app = express();
 
+/**
+ * Устанавливаем pug, как шаблонизатор по умолчанию и указываем путь до шаблонов.
+ */
+app.set(`views`, path.resolve(__dirname, PATH_TO_TEMPLATES_DIR));
+app.set(`view engine`, `pug`);
+
+
 /**
  * Подключаем Router'ы для путей.
  */
 app.use(`/`, rootRouter);
 app.use(`/my`, myRouter);
 app.use(`/articles`, articlesRouter);
 
+/**
+ * Добавляем отдачу статичных файлов.
+ */
+app.use(express.static(path.resolve(__dirname, PATH_TO_PUBLIC_DIR)));
+
+
+/**
+ * Добавляем обработчики ошибок.
+ */
+app.use((req, res) => res.status(404).render(`errors/404`));
+
+app.use((err, req, res, _next) => {
```

Comment:
если _next не вызывается, то и объявлять его не нужно

Reply:
не соглашусь, чтобы была обработка ошибок express требует 4 аргумента, по их количеству он и определяет, что это обработка 500 ошибки:
```Define error-handling middleware functions in the same way as other middleware functions, except error-handling functions have four arguments instead of three: (err, req, res, next).```
http://expressjs.com/en/guide/error-handling.html

Reply:
да, действительно, давно уже с express не работал. Сейчас кстати очень популярен https://docs.nestjs.com/
В свободное время можешь поизучать.